games/m.yaml

```diff
@@ -386,6 +386,26 @@
   - https://camo.githubusercontent.com/897d01a707f6a117284446dffd45998c3a98aaef/687474703a2f2f692e696d6775722e636f6d2f6d7a557771506f2e706e67
   video:
     youtube: Xq3isov6mZ8
+    
+- name: minecraft-weekend
+  originals:
+  - Minecraft
+  type: clone
+  repo: 'https://github.com/jdah/minecraft-weekend'
+  development: active
+  status: playable
+  lang:
+  - C
+  framework:
+  - OpenGL
+  license:
+  - MIT
+  updated: '2020-05-10'
+  images:
+  - 'https://github.com/jdah/minecraft-weekend/blob/master/screenshots/1.png'
```

Comment:
this link doesn't lead to an image file; normally you need to click the Raw button to get to the image link but I see that this file is too big so github serves it as a download instead.